tests/failing/bad_json

```diff
@@ -0,0 +1 @@
+<json>This isn't json</json>
```

Comment:
for all the `failing/*` and `passing/*` files, please append `.json`. This way, when the file is loaded in any editor, it can highlight syntax properly for json